src/mono/mono/mini/mini-arm.c

```diff
@@ -6451,18 +6434,16 @@ mono_arch_emit_prolog (MonoCompile *cfg)
 			}
 			case RegTypeStructByVal: {
 				int doffset = inst->inst_offset;
-				int soffset = 0;
+				//int soffset = 0;
```

Comment:
I kept this one commented since it is referenced in a commented line with a FIXME below.